handbook/people-ops/buddy-program.md

```diff
@@ -0,0 +1,36 @@
+# Onboarding buddy program
+
+
+## Why we have onboarding buddies
+To ensure our new teammates are set up for success, we will pair them up with an “onboarding buddy” in order to become better acquainted with Sourcegraph team and culture. An onboarding buddy is an additional resource for new teammates who are learning how to navigate their new role and company.
+
+An onboarding buddy is essentially a new teammate’s first friend– someone to connect with that can share their experiences and advice to help make their transition a little smoother. A buddy will be the new teammate’s go-to person for questions (in addition to their manager and the People Ops team). They are someone to turn to with questions, someone who knows the ins and outs of the company and can offer help and guidance as the new teammate navigates their first few weeks at Sourcegraph. 
+
+## A great onboarding buddy will be
+**Approachable:** the new teammate is probably overwhelmed, let them know you can be accessed along with their manager if they have any questions or concerns along the way.
+
+**Proactive:** check-in and see how they’re doing, if they need help with anything. 
+
+**Compassionate:** remember what it was like to be new and tries to help ease that transition.
+
+**Helpful:** shows them the ropes of the organization and gives them tips on navigating it. 
+
+
+
+## Onboarding buddy logistics
+**Total length**
+
+- 3 months
+
+**Frequency**
+- Month 1: 1x per week for 25 min.
+ 
+- Month 2: 1x every two weeks for 25 min.
+
+- Month 3: 1x a month for 25 min.
+
+
+## Buddy selection
```

Comment:
Anything about similar level of seniority?

Reply:
Also, how long should someone be on the team before they become eligible to be a buddy themselves? Just once they graduate from their own buddy program?

Reply:
Good points! We've had folks on the exec team be buddies recently and I don't want to restrict anyone by seniority for now– but will keep in mind as we scale! 

I did add a bullet point for eligibility (3 months, so as soon as they've completed their buddy program).